packages/squiggle-lang/src/rescript/Utility/XYShape.res

```diff
@@ -453,6 +453,41 @@ module PointwiseCombination = {
     T.filterOkYs(newXs, newYs)->Ok
   }
 
+  // Nuño wrote this function to try to increase precision, but it didn't work.
```

Comment:
I really don't like having much dead code around. At least give a longer comment describing it some more, and flagging that it's not being used now.